relayer/src/connection.rs

```diff
@@ -33,16 +32,45 @@ pub use error::ConnectionError;
 /// Maximum value allowed for packet delay on any new connection that the relayer establishes.
 pub const MAX_PACKET_DELAY: Duration = Duration::from_secs(120);
 
-pub fn from_retry_error(e: retry::Error<ConnectionError>, description: String) -> ConnectionError {
-    match e {
-        retry::Error::Operation {
-            error,
-            total_delay,
-            tries,
-        } => {
-            ConnectionError::max_retry(description, tries, total_delay, Box::new(error.0))
+// Includes utility methods and constants to configure the retry behavior
+// for the channel handshake algorithm.
```

Comment:
I think the way to properly document this sub-module would be to move these lines inside and at the top of the module, as well as change them to `//!` doc strings.

Reply:
Got it! 9191e2ee

Reply:
By the way, I hesitated originally to make doc comments because `mod handshake_retry` is supposed to be a private module. I guess even private module should do `//!` doc string then, right?

Reply:
I don't believe adding doc strings makes the module public. Documentation is still super useful for internal developers 😄